tests/func/test_args.py

```diff
@@ -13,6 +13,7 @@ def code_to_debug():
         import sys
         from dbgimporter import import_and_enable_debugger
         import_and_enable_debugger()
+        print(sys.argv)
```

Comment:
print for debug?

Reply:
Sorry, didn't notice this before. Yes, it's diagnostic, but since it's a test, it doesn't hurt to have it here - this way we get the whole thing in the log if it fails in an automated run.